packages/x64dbg.dbgchild.vm/x64dbg.dbgchild.vm.nuspec

```
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>x64dbg.dbgchild.vm</id>
+    <version>10.0.0</version>
```

tricky though, as it is 10 beta. 
```suggestion
    <version>10</version>
```

[nitpick] I would change to 10, as I think the tool version does not use the second and third segments. This is